mlos_bench/mlos_bench/config/services/remote/azure/scripts/setup-rg/README.md

```diff
@@ -0,0 +1,68 @@
+# Resource Group Setup
+
+These scripts are helpers for setting up a new resource group (RG) in Azure as the *control plane* for MLOS.
+The *control plane RG* is a container for the *persistent* resources of MLOS (results/metrics storage, scheduler VM, notebook interface, etc.).
+
+## Quickstart
+
+1. Starting in this current directory, ensure that we are logged in to Azure CLI.
+
+    ```sh
+    az login
+    ```
+
+1. Make a copy of the control plane ARM parameters file.
```

Comment:
The numbering on these are all `1.` which won't render well.  Please have a look in the markdown linter inside the devcontainer.

Reply:
Changed back to actual numbers